github/release_notes.py

```diff
@@ -14,26 +14,81 @@
 
 import re
 from collections import namedtuple
-
 import git
 from git.exc import GitError
 from pydash import _
 from semver import parse_version_info
 from github.util import GitHubRepositoryHelper
 
 from util import info, warning, fail, verbose, existing_dir
+from product.model import ComponentReference
+from model.base import ModelValidationError
+
+ReleaseNote = namedtuple('ReleaseNote', [
+    "category_id",
+    "target_group_id",
+    "text",
+    "reference_is_pr",
+    "reference_id",
+    "user_login",
+    "origin_repo",
+    "is_current_repo",
+    "component_ref"
+])
+
+def create_release_note_obj(
+    category_id: str,
+    target_group_id: str,
+    text: str,
+    reference_is_pr: str,
+    reference_id: str,
+    user_login: str,
+    origin_repo: str,
+    is_current_repo: bool
+)->ReleaseNote:
+
+    if reference_id:
+        reference_id=str(reference_id)
+
+    return ReleaseNote(
+        category_id=category_id,
+        target_group_id=target_group_id,
+        text=text,
+        reference_is_pr=reference_is_pr,
+        reference_id=reference_id,
+        user_login=user_login,
+        origin_repo=origin_repo,
+        is_current_repo=is_current_repo,
+        component_ref=ComponentReference.create(name=origin_repo, version=None)
```

Comment:
hmm.. I suppose it would make sense to also introduce a separate type representing only the ComponentName (maybe we should even call it like that)